src/relay/backend/build_module.cc

```diff
@@ -433,9 +435,14 @@ class RelayBuildModule : public runtime::ModuleNode {
     // Get the updated function.
     auto func = Downcast<Function>(relay_module->Lookup("main"));
 
+    // Check whether to use TOPI schedule.
```

Comment:
Can we do this in compile_engine instead of here? It seems that we just pass the extracted flag to graph_runtime_codegen and then to compile_egine.